resources/dicts/relationship_events/cat_to_other/apprentice_to_other.json

```diff
@@ -98,7 +99,8 @@
             "wishes they had (cat)'s mentor instead.",
             "offers to help (cat) clean out the elder's bedding.",
             "promises not to tell (cat)'s mentor that they've been sneaking out of camp.",
-            "playfully teases (cat) for missing a catch."
+            "playfully teases (cat) for missing a catch.",
+            "promises (cat) that they'll be mates when they become warriors."
```

Comment:
I know there is not currently the possibility for exact romantic interaction between apprentices, so I would suggest removing this string. Because not every close apprentices want to have a mate-relationship with each other.